turbo.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://turbo.build/schema.json",
-  "pipeline": {
+  "tasks": {
```

Comment:
You can refer to this [link](https://turbo.build/repo/docs/reference/turbo-codemod#rename-pipeline).